File: app/src/androidTest/java/com/ndmsystems/coala/TestHelper.java
package com.ndmsystems.coala;

import com.ndmsystems.infrastructure.logging.ILogger;

/**
 * Created by bas on 16.12.16.
 */

public class TestHelper implements ILogger {
    @Override
    public void v(String message) {
        System.out.println(message);
    }

    @Override
    public void d(String message) {
        System.out.println(message);
    }

    @Override
    public void i(String message) {
        System.out.println(message);
    }

    @Override
    public void w(String message) {
        System.out.println(message);
    }

    @Override
    public void e(String message) {
        System.out.println(message);
    }
}

File: app/src/test/java/com/ndmsystems/coala/TestHelper.kt
package com.ndmsystems.coala

import com.ndmsystems.coala.helpers.logging.ILogger

class TestHelper : ILogger {
    override fun v(message: String) {
        println(message)
    }

    override fun d(message: String) {
        println(message)
    }

    override fun i(message: String) {
        println(message)
    }

    override fun w(message: String) {
        println(message)
    }

    override fun e(message: String) {
        println(message)
    }
}